optionals/nodefaultactionicon/CfgInGameUI.hpp

```diff
@@ -0,0 +1,5 @@
+class CfgInGameUI {
+    class DefaultAction {
+        size = "0";
```

Comment:
this could be something like
`size = "profileNamespace getVariable ['actionSizeSetting', 0.025];";`
but it seems to require full game restart to take effect

Reply:
Yep, this is something that could be a client / local only setting (not forcable). It would require a game restart, but that is still a million times better than optional pbos.
It may need a dummy entry, because numbers may not be read correctly if the expression contains a comma. This needs more research though and might not apply to all config entries.
The helper technique I posted on Slack once.

Reply:
What an old thing revived! :+1: 

Would definitely be great, going to leave it to @commy2 or someone else to test though, I probably won't have time that soon.

Reply:
FYI, the changes in CfgInGameUI break BIS_fnc_holdActionAdd (the hold action introduced by APEX), the symbol is not shown.

Reply:
You mean this pull request or something else in ACE?

Reply:
The code referenced above
```class CfgInGameUI {
    class DefaultAction {
        size = "0";```